### ISSUE TITLE
b=text where text is too long

### ISSUE THREAD

**Issue by H077y** [systemic-chaos/currently-fronting#206]
When `b=text` is used, and the text in the box is too long to display in that section, it will underlap under the description box.\
In Felix's case, `— Meow/Meows` has been hidden from the bottom of his pronouns section as it's hidden underneath the description field.\
![image](https://user-images.githubusercontent.com/64114013/206908846-90b73df2-282d-43bc-9b43-c00e480b61b5.png)\
![image](https://user-images.githubusercontent.com/64114013/206909005-76f03025-513e-40a9-9dd8-8132c77dd98b.png)

**Comment by ArcanaOfSouls** [comment]
Hm. That... Is unintended. We don't plan to increase the maximum size of banners for performance reasons, but we *should* be able to figure out how to get the text in the banner to overflow the banner's bounding box without being hidden.

Thanks for the heads up! We'll take a look. :)

**Comment by H077y** [comment]
No problem :3 I look forward to seeing what you decide to do.

**Comment by ArcanaOfSouls** [comment]
@H077y Gonna try to address this now. You probably already noticed, but we took care of #193 as of a few days ago. Feel free to mention us as @ArcanaOfSouls on that issue if for some reason it isn't working, though. (We're taking the lack of response thus far as a good sign.)

**Comment by H077y** [comment]
I keep forgetting to check that one, sorry for not responding. I wanted to test it out then send back how it went, but I'm yet to do that. I'll try it in about 30 minutes.
I look forward to seeing the progress on this :3

**Comment by ArcanaOfSouls** [comment]
Excellent! And no need to apologize! We just wanted to be sure you saw. We have issues on this repo configured to automatically close when their corresponding PR is merged (although that one glitched and was closed a bit later manually, LOL). So if you're not watching the repo, you may not be aware of bugfixes or enhancements.

Also, we went ahead and invited you to this project with Write access, meaning you should be able to create branches here and submit your own PRs from within the repository itself if you so choose. If you choose to accept the invite, you'll be able to involve yourself a bit more closely with the codebase. No worries if you're not interested though!

You just happen to be, by far, the user to notice more edge cases than any other. And we appreciate that kind of insight in any case, whether a member of the project or not.

We're also in the process of migrating this app to a new hosting platform. That's actually our priority for the project once this last bug is squashed. And as part of that migration, the app will be available at a new, shorter domain name for ease of linking while confined to the limitations of Discord's maximum profile character length. We'll file a separate issue to track progress on that. :3

**Comment by H077y** [comment]
I forgot to check it after I got on the PC, again. But thankfully I caught your update before I had time to miss it.
I've just checked out the date update and it's working perfectly by the looks of it.

And sure! I'd be happy to help if I find any issues. I hope I can be of help :3

**Comment by ArcanaOfSouls** [comment]
I apologize. Just realized that granting those permissions granted the ability to close all PRs and issues, even if only Triage permissions are granted. We were exhausted yesterday and somehow didn't think things through clearly. That kind of power makes us nervous given how much time we've spent on this and how important it is to us. Had to undo the permission grant. Still, if you have a feature you'd like to see added (or simply see a bugfix issued more quickly), we do accept PRs from forks.

Thanks for understanding. And thanks for the continued interest!

To be completely honest, we are likely abandoning use of Discord almost altogether soon. We've always had some issues with their lack of privacy-preserving tech and lack of policies protecting minority users (like systems) from harassment. And their recent Android app updates make the app almost unuseable for us; and we've relied on it for system management on-the-go.

Don't worry, though! This project isn't going anywhere. In fact, when support for installation as a PWA is enabled... Well, let's just say: We've long wanted to see this app work with or without a Discord account. Stay tuned!

**Comment by H077y** [comment]
I understand completely, don't worry :3

Also, agreed. The new android Discord update is unusable. Our main phone is on an outdated version (127.9 Beta) before the react native update, and we leave the horrible new update on a second phone. Though that does mean we have to suffer through it at night.

**Comment by ArcanaOfSouls** [comment]
@H077y 
> The new android Discord update is unusable. Our main phone is on an outdated version (127.9 Beta) before the react native update, and we leave the horrible new update on a second phone. Though that does mean we have to suffer through it at night.

ooooo. You& will almost certainly appreciate our in-progress plan to enable an alternative sign-in method---and system management, as a result---through a platform entirely independent of Discord or any other nonfree software. :3

Don't worry, PK support isn't going anywhere. We just want to allow people to avoid Discord entirely if they can. And that of course means supporting an API other than that exposed by PluralKit; which, as a Discord bot, depends on Discord to operate. Stay tuned...

**Comment by H077y** [comment]
Perhaps y'all could use the SimplyPlural API? That's disconnected from Discord, though it's not as simple to use from what I've heard and seen.

**Comment by ArcanaOfSouls** [comment]
> Perhaps y'all could use the SimplyPlural API? That's disconnected from Discord, though it's not as simple to use from what I've heard and seen.

SimplyPlural is nonfree copyright-reserved software with source available. We have serious concerns with the way it is operated. Which we expressed to its creators, who updated their documentation to indicate that it was not truly open source software (whereas before their site listed it as open source, which was entirely inaccurate). We were disappointed that it wasn't made open source at that point.

**Comment by ArcanaOfSouls** [comment]
**TL;DR**: We will not be implementing that API. This is something altogether separate.

> enable an alternative sign-in method---and system management, as a result---through a platform entirely independent of Discord or any other nonfree software

You have not seen this platform because we are in the process of creating it ourselves. Stay tuned. :3

(And don't worry. We will get this PR completed very soon.)
